### PR TITLE
closes #63 closes #65 closes #66 closes #67 extensions

### LINKED ISSUE #63: Documentation:  Clarify where to put TA specific assertions
### Please select a category the issue is focused on?

Other

### Let us know where something needs a refresh or put your idea here!

https://pharmaverse.github.io/admiraldev/main/articles/admiraldev.html#scenario-two-admiral-extension

Update point 2 so that clear that if an assertion cannot be generalised beyond a specific TA then it should remain in admiralonco for example, e.g. see signal_crpr

### LINKED ISSUE #65: General Issue: Updates imports to align with admiral
### Background Information

Description fIle for admiraldev needs to be updated to mimic admiral

### Definition of Done

Description fIle for admiraldev needs to be updated to mimic admiral

### LINKED ISSUE #66: Documentation:  Change Functions in Navbar back to References
### Please select a category the issue is focused on?

Other

### Let us know where something needs a refresh or put your idea here!

Change Functions in Navbar back to References

### LINKED ISSUE #67: Documentation:  Typo in Description File
### Please select a category the issue is focused on?

Other

### Let us know where something needs a refresh or put your idea here!

image

### PR DESCRIPTION
**Pull request by rossfarrugia**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page.
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging.
- [x] Address all merge conflicts and resolve appropriately.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@bms63 i had one of these assigned to me so i just picked up a small bundle of issues to update in one go. seems the R CMD check fails due to admiral.test though - any idea here? not sure why this didn't affect the other PRs before this one. also is it OK the code coverage is so low for this package?

**Comment by bms63** [comment]
> @bms63 i had one of these assigned to me so i just picked up a small bundle of issues to update in one go. seems the R CMD check fails due to admiral.test though - any idea here? not sure why this didn't affect the other PRs before this one. also is it OK the code coverage is so low for this package?

I think a goal for this release cycle is to boost the test coverage for this package.  admiral went from 91% to 97% test coverage with moving these functions over to `admiraldev` so I think a lot of our coverage issues centered around our development utilities.

File: staged_dependencies.yaml [hunk 1]
```diff
@@ -3,6 +3,8 @@ current_repo:
   repo: pharmaverse/admiraltemplate
   host: https://github.com
 upstream_repos:
+  - repo: pharmaverse/admiral.test
+    host: https://github.com
   - repo: pharmaverse/admiral
     host: https://github.com
```
Comment (thomas-neitmann) [inline staged_dependencies.yaml:9]: Actually admiral is a downstram repo to admiraldev.

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]

**Review by bms63 — APPROVED** [review]
Thanks Ross.

File: staged_dependencies.yaml (hunk 1 above)
Comment (rossfarrugia) [reply staged_dependencies.yaml]: good spot Thomas - now removed

**Review by thomas-neitmann — APPROVED** [review]

**Comment by rossfarrugia** [comment]
yes i asked @bms63 if he'd have time to fix these as same will be in other admiral packages too and i can focus on onco release then - it's already fixed in onco.

**Pull request merged**